extensions/stackdriver/log/logger.cc

```diff
@@ -243,8 +255,19 @@ bool Logger::exportLogEntry(bool is_on_done) {
 
 void Logger::addTCPLabelsToLogEntry(
     const ::Wasm::Common::RequestInfo& request_info,
+    const ::Wasm::Common::FlatNode& peer_node_info,
     google::logging::v2::LogEntry* log_entry) {
   auto label_map = log_entry->mutable_labels();
+  setSourceCanonicalService(peer_node_info, label_map);
+  log_entry->set_text_payload(absl::StrCat(
+      label_map->find("source_canonical_service") != label_map->end()
```

Comment:
This looks up map two times. Better get iter beforehand and look up map for only one time.

Reply:
Thanks..Fixed